js/imagediff.js

```diff
@@ -343,16 +374,16 @@
     toImageData : function (object) {
       checkType(object);
       if (isImageData(object)) { return copyImageData(object); }
-      return toImageData(object);
+      return toImageData.apply(null, arguments);
     },
 
-    equal : function (a, b, tolerance) {
+    equal : function (a, b, tolerance, callback) {
       checkType(a, b);
       a = toImageData(a);
       b = toImageData(b);
       return equal(a, b, tolerance);
     },
-    diff : function (a, b) {
+    diff : function (a, b, callback) {
```

Comment:
This and the following line:
https://github.com/HumbleSoftware/js-imagediff/pull/26/files#diff-072ad2b67177ebe639260e22ca6f414dR116

We need to load the base64 src to an image before drawing that image to a canvas.